src/memote/__init__.py

```diff
@@ -30,6 +30,18 @@
 from memote.suite.reporting import *
 from memote.jinja2_extension import *
 
+import sys
+import logging
+
 __author__ = "Moritz E. Beber"
 __email__ = "morbeb@biosustain.dtu.dk"
 __version__ = "0.9.12"
+
+LOGGER = logging.getLogger()
+
+if sys.version_info < (3,):
+    LOGGER.warning('You are using `Memote` with Python 2. '
```

Comment:
Rather than logging a warning I would raise a `FutureWarning`.

Reply:
It doesn't look like this check runs when memote is installed with `pip > 9.0` AND `python_requires = >=3.5`. In that case, pip will actually already print the following error when installing memote:

```
ERROR: Package u'memote' requires a different Python: 2.7.9 not in '>=3.5,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*'
```

Reply:
But I suppose it would if users manage to install memote with an older version of pip.